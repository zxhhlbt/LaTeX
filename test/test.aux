\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\bibstyle{plain}
\HyPL@Entry{0<</S/D>>}
\HyPL@Entry{3<</P(\376\377\040\023\000\040\000I\000\040\040\023)>>}
\HyPL@Entry{4<</P(\376\377\040\023\000\040\000I\000I\000\040\040\023)>>}
\HyPL@Entry{5<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {第1节}Section1}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}SubSection1}{1}{subsection.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.1.1}SubSubSection1}{1}{subsubsection.1.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {第2节}Section2}{1}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}SubSection2}{1}{subsection.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1}SubSubSection2}{1}{subsubsection.2.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {第3节}Section3}{1}{section.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}SubSection3}{1}{subsection.3.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.1.1}SubSubSection3}{1}{subsubsection.3.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {第4节}Cornering}{1}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}sub}{1}{subsection.4.1}\protected@file@percent }
\citation{冯黎-118}
\citation{薛博元方艳-119}
\citation{谌华郭伟-117}
\citation{谢晶-116}
\citation{贾会群-120}
\bibdata{reference/re}
\bibcite{冯黎-118}{1}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces table1\relax }}{2}{table.caption.4}\protected@file@percent }
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{表格1}{{1}{2}{table1\relax }{table.caption.4}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces table2\relax }}{2}{table.caption.5}\protected@file@percent }
\bibcite{薛博元方艳-119}{2}
\bibcite{谌华郭伟-117}{3}
\bibcite{谢晶-116}{4}
\bibcite{贾会群-120}{5}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces figure1\relax }}{3}{figure.caption.6}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces figure2\relax }}{3}{figure.caption.7}\protected@file@percent }
\@writefile{lof}{\contentsline {subfigure}{\numberline{(a)}{\ignorespaces {12321}}}{3}{subfigure.2.1}\protected@file@percent }
\@writefile{lof}{\contentsline {subfigure}{\numberline{(b)}{\ignorespaces {32123}}}{3}{subfigure.2.2}\protected@file@percent }
