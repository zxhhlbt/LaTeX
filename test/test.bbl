\begin{thebibliography}{1}

\bibitem{冯黎-118}
冯黎.
\newblock {\em 无人驾驶汽车自主导航策略与控制算法研究}.
\newblock PhD thesis, 电子科技大学, 2019.
\newblock 期: 12.

\bibitem{薛博元方艳-119}
薛博元 and 方艳.
\newblock 无人驾驶汽车道路检测与识别方法研究.
\newblock {\em 南方农机}, (1):207, 2019.

\bibitem{谌华郭伟-117}
谌华, 郭伟, 闫敬文, 卓文浩, and 吴良斌.
\newblock 基于深度学习的sar图像道路识别新方法.
\newblock {\em 吉林大学学报(工学版)}, pages 1--10, 2019.
\newblock 页数: 10.

\bibitem{谢晶-116}
谢晶.
\newblock 基于机器视觉的道路识别技术专利布局与发展趋势.
\newblock {\em 现代信息科技}, 3(06):174--177, 2019.
\newblock 页数: 4.

\bibitem{贾会群-120}
贾会群.
\newblock {\em 无人驾驶车辆自主导航关键技术研究}.
\newblock PhD thesis,
  中国科学院大学(中国科学院长春光学精密机械与物理研究所),
  2019.
\newblock 期: 08.

\end{thebibliography}
