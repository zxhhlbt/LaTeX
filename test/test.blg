This is BibTeX, Version 0.99d (TeX Live 2019/W32TeX)
Capacity: max_strings=100000, hash_size=100000, hash_prime=85009
The top-level auxiliary file: test.aux
The style file: plain.bst
Database file #1: reference/re.bib
Warning--there's a number but no volume in 薛博元方艳-119
You've used 5 entries,
            2118 wiz_defined-function locations,
            526 strings with 4644 characters,
and the built_in function-call counts, 1463 in all, are:
= -- 147
> -- 62
< -- 1
+ -- 25
- -- 20
* -- 95
:= -- 268
add.period$ -- 19
call.type$ -- 5
change.case$ -- 23
chr.to.int$ -- 0
cite$ -- 6
duplicate$ -- 56
empty$ -- 106
format.name$ -- 20
if$ -- 298
int.to.chr$ -- 0
int.to.str$ -- 5
missing$ -- 3
newline$ -- 32
num.names$ -- 10
pop$ -- 16
preamble$ -- 1
purify$ -- 20
quote$ -- 0
skip$ -- 38
stack$ -- 0
substring$ -- 70
swap$ -- 10
text.length$ -- 1
text.prefix$ -- 0
top$ -- 0
type$ -- 20
warning$ -- 1
while$ -- 14
width$ -- 6
write$ -- 65
(There was 1 warning)
