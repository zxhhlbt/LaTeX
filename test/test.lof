\contentsline {figure}{\numberline {1}{\ignorespaces figure1\relax }}{3}{figure.caption.6}% 
\contentsline {figure}{\numberline {2}{\ignorespaces figure2\relax }}{3}{figure.caption.7}% 
\contentsline {subfigure}{\numberline {(a)}{\ignorespaces {12321}}}{3}{subfigure.2.1}% 
\contentsline {subfigure}{\numberline {(b)}{\ignorespaces {32123}}}{3}{subfigure.2.2}% 
