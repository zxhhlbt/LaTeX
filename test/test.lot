\contentsline {table}{\numberline {1}{\ignorespaces table1\relax }}{2}{table.caption.4}% 
\contentsline {table}{\numberline {2}{\ignorespaces table2\relax }}{2}{table.caption.5}% 
