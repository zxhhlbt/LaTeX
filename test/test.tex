%----------------------------------
%-------------导言区----------------
\documentclass{zxhClass}
\usepackage{zxhMacro}
\usepackage{zxhMacroset}

%----------------------------------
%-------------正文区----------------
\begin{document}
\cfoot{}
\input{doc/cover.tex}      %封面
\newpage
\input{doc/copyright.tex}  %版权
\newpage
\input{doc/titlepage.tex}  %扉页
\newpage
%---------\pagenumbering{Roman}----------
\cfoot{\thepage}
\renewcommand{\thepage}{--~\Roman{page}~--}     
\setcounter{page}{1} 
\input{doc/abstract.tex}   %摘要
\newpage

\tableofcontents
\listoftables   
\listoffigures  
\newpage
%------------------BODY------------------
\renewcommand{\thepage}{\arabic{page}}      
\setcounter{page}{1}

\input{doc/section1.tex}  
\input{doc/section2.tex}
\input{doc/section3.tex}

\input{onlytest.tex}	

%----------------附录----------------
% \appendixname
% \input{doc/appendix.tex}

\cite{冯黎-118}       %在正文中插入
\cite{薛博元方艳-119}
\cite{谌华郭伟-117}
\cite{谢晶-116}
\cite{贾会群-120}

%----------------参考文献-----------------
\bibliography{reference/re}

\end{document}
