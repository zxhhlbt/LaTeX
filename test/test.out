\BOOKMARK [1][-]{section.1}{\376\377\000S\000e\000c\000t\000i\000o\000n\0001}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0001}{section.1}% 2
\BOOKMARK [3][-]{subsubsection.1.1.1}{\376\377\000S\000u\000b\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0001}{subsection.1.1}% 3
\BOOKMARK [1][-]{section.2}{\376\377\000S\000e\000c\000t\000i\000o\000n\0002}{}% 4
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0002}{section.2}% 5
\BOOKMARK [3][-]{subsubsection.2.1.1}{\376\377\000S\000u\000b\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0002}{subsection.2.1}% 6
\BOOKMARK [1][-]{section.3}{\376\377\000S\000e\000c\000t\000i\000o\000n\0003}{}% 7
\BOOKMARK [2][-]{subsection.3.1}{\376\377\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0003}{section.3}% 8
\BOOKMARK [3][-]{subsubsection.3.1.1}{\376\377\000S\000u\000b\000S\000u\000b\000S\000e\000c\000t\000i\000o\000n\0003}{subsection.3.1}% 9
\BOOKMARK [1][-]{section.4}{\376\377\000C\000o\000r\000n\000e\000r\000i\000n\000g}{}% 10
\BOOKMARK [2][-]{subsection.4.1}{\376\377\000s\000u\000b}{section.4}% 11
